zrml/hybrid-router/src/tests/buy.rs

```diff
@@ -673,7 +673,7 @@ fn buy_fails_if_balance_too_low() {
         let asset = Assets::CategoricalOutcome(market_id, 0);
         let amount = 10 * BASE;
 
-        assert_ok!(Balances::set_balance(RuntimeOrigin::root(), ALICE, amount - 1, 0));
+        assert_eq!(Balances::set_balance(&ALICE, amount - 1), amount - 1);
```

Comment:
Alice is able to set the balance?

Reply:
No, the `ALICE` object is just the address whose balance is set. The `set_balance` call is not publicly exposed API.